packages/ui-docs-client/src/index.js

```diff
@@ -47,7 +47,7 @@ function renderDocsClient(data, element) {
 
   ReactDOM.render(
     <EmotionThemeProvider theme={canvas}>
-      <App {...data} />,
```

Comment:
this was a bug from a previous commit that caused the nav layout to shift up a bit